core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -56,6 +56,7 @@
     public static final String OPTIMIZE_HASH_GENERATION = "optimize_hash_generation";
     public static final String JOIN_DISTRIBUTION_TYPE = "join_distribution_type";
     public static final String JOIN_MAX_BROADCAST_TABLE_SIZE = "join_max_broadcast_table_size";
```

Comment:
yes, I'll keep that for a follow-up PR though since we haven't broken up SystemSessionProperties so far even though there are other examples of separate Configs